clouddriver-kubernetes/src/main/groovy/com/netflix/spinnaker/clouddriver/kubernetes/v1/security/KubernetesV1Credentials.java

```diff
@@ -131,8 +131,11 @@ private void configureDockerRegistries() {
 
   public List<String> getDeclaredNamespaces() {
     if (namespaces != null && !namespaces.isEmpty()) {
+      List<String> toInitialize = new ArrayList<String>(namespaces);
```

Comment:
The `ArrayList` can omit the type parameter (`ArrayList<>(namespaces)`)

Reply:
👍